File: GameScrubsV2/Models/BracketPosition.cs
using GameScrubsV2.Enums;

namespace GameScrubsV2.Models;

public partial class BracketPosition
{
	public int Id { get; set; }

	public BracketType Type { get; set; }

	public string? Player1 { get; set; }

	public string? Player2 { get; set; }

	public string? WinLocation { get; set; }

	public string? LoseLocation { get; set; }
}

File: GameScrubsV2.Web/src/models/BracketPosition.ts
import type { BracketType } from './BracketType';

export interface BracketPosition {
  id: number;
  type: BracketType;
  player1: string | null;
  player2: string | null;
  winLocation: string | null;
  loseLocation: string | null;
  markerPosition: number | null;
}
